### PR TITLE
New user invite

### PR DESCRIPTION
**Pull request by Justin-Holmes**

### CODE REVIEW

File: lib/ello/mailchimp_wrapper.rb [hunk 1]
```diff
@@ -18,14 +18,15 @@ def remove_from_users_list(email)
...
-  def upsert_to_users_list(email, preferences, categories = [])
+  def upsert_to_users_list(email, preferences, categories = [], has_account = true)
     return if skip_list.include?(email)
     hash = subscriber_hash(email)
     begin
       users_list.members(hash).upsert(
         body: {
           email_address: email,
           status_if_new: 'subscribed',
+          has_account: has_account,
```
Comment (jayzes) [inline lib/ello/mailchimp_wrapper.rb]: Do we need to add this attribute in Mailchimp somewhere before using it?

File: spec/ello/kinesis_consumer/knowtify_processor_spec.rb [hunk 2]
```diff
@@ -18,6 +18,34 @@
...
+
+      it 'creates a record in Knowtify' do
+        expect_any_instance_of(Knowtify::Client).to receive(:upsert).with([{
+          email: 'test@example.com',
+          data: {
+            subscribed_to_users_email_list: true,
+            subscribed_to_invitation_drip: true,
+            subscribed_to_daily_ello: true,
+            subscribed_to_weekly_ello: true
+          }
```
Comment (jayzes) [inline spec/ello/kinesis_consumer/knowtify_processor_spec.rb]: Should we add an analogous `has_account` to Knowtify as well?

File: lib/ello/mailchimp_wrapper.rb (hunk 1 above)
Comment (Justin-Holmes) [inline lib/ello/mailchimp_wrapper.rb]: Yep we do, I've already added it so we should be good.

**Comment by Justin-Holmes** [comment]
@jayzes Added the additional pieces and everything looks good on staging 👍

**Pull request merged**